### PR TITLE
Update graphDb to 9.4.0 and fix tests

### PR DESCRIPTION
**Pull request by teodossidossev**
With changes in the rdf4j (https://github.com/eclipse/rdf4j/pull/2414) the ArrangedWriter class was deprecated.

As a side effect of this change now TurtleWriter-based exports have all the namespaces that are in the repository regardless of whether they are used by any of the exported statements.

This leads to the need to change the tests, although the breaking change is marked as a bug (https://github.com/eclipse/rdf4j/issues/2694).

### CODE REVIEW

**Review by svilenvelikov — DISMISSED** [review]

File: test-e2e/tests/rdfstar.spec.js [hunk 1]
```diff
@@ -111,9 +107,11 @@ describe('Should test RDFStar', () => {
...
     return rdfClient.query(payload).then((resp) => {
-      resp.on('data', data => {
-        expect(data.toString().trim()).toEqual(expected.trim());
-      });
+      resp.once('data', onData);
     });
   });
 });
+
+function onData(data) {
```
Comment (svilenvelikov) [inline test-e2e/tests/rdfstar.spec.js:115]: Move this inside the spec

Comment (teodossidossev) [reply test-e2e/tests/rdfstar.spec.js]: done

**Comment by svilenvelikov** [comment]
> This leads to the need to change the tests, although the breaking change is marked as a bug (eclipse/rdf4j#2694)

So, regarding this, we should watch this issue and when it gets fixed and GDB gets that version, we should update the library as well. I'll create an issue for this.

**Review by svilenvelikov — DISMISSED** [review]

File: test-e2e/tests/rdfstar.spec.js [hunk 2]
```diff
@@ -111,9 +114,19 @@ describe('Should test RDFStar', () => {
...
-      resp.on('data', data => {
-        expect(data.toString().trim()).toEqual(expected.trim());
-      });
+      expected = Utils.loadFile('./data/rdfstar/expected_response_sparql_tsv_star.txt');
+      response = resp;
+      response.on('data', onData);
     });
   });
+
+  function onData(data, resp) {
```
Comment (svilenvelikov) [inline test-e2e/tests/rdfstar.spec.js:123]: You don't have a `resp` arg here.

Comment (teodossidossev) [reply test-e2e/tests/rdfstar.spec.js]: done

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**